### ISSUE TITLE
Reply mention off still ping

### ISSUE THREAD

**Issue by Cope-Harder** [Aliucord/plugin-requests#262]
### Discord Account

Ol4ax#5081

### Plugin Description

When someone replies to your message but they turn off the ping, make it still ping me so I know they replied 

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by scurh** [comment]
that.. ruins the point of having ping turned off.

**Comment by hairyfred** [comment]
> 
> 
> that.. ruins the point of having ping turned off.

He wants all replys to be pings
even if the user selects not to ping
some people prefere the pings

**Comment by scurh** [comment]
> > that.. ruins the point of having ping turned off.
> 
> He wants all replys to be pings
> even if the user selects not to ping
> some people prefere the pings

yes i understand that but it seems pointless, just ask them to mention you.

**Comment by hairyfred** [comment]
> 
> 
> > > that.. ruins the point of having ping turned off.
> > 
> > 
> > He wants all replys to be pings
> > even if the user selects not to ping
> > some people prefere the pings
> 
> yes i understand that but it seems pointless, just ask them to mention you.

Its a plugin request
clearly someone has an idea/feature they want so they request it.
its up to a dev if they want to make it
some plugins are pointless for some, usefull for others

**Comment by scurh** [comment]
i guess, i was just speaking my mind

**Comment by rushiiMachine** [comment]
this will be purely visual, push notifications still wont work
